### PR TITLE
feat(ft): add twig support

### PR DESCRIPTION
**Pull request by gbprod**
https://twig.symfony.com/doc/3.x/templates.html#comments

### CODE REVIEW

**Comment by numToStr** [comment]
Is `html.twig` set by nvim? And what about syntax highlighting?

**Comment by gbprod** [comment]
When I open `twig` files (without any plugins), neovim set `html.twig` as filetype. I can't find why looking at Nevim source code...
If I set `twig`as filetype in `ft.lua`, it doesn't work at all.

Currently, I use `commentstring` to make your plugin works on twig files.

Syntax highlight doesn't work without plugins in Neovim but I've [made a PR](https://github.com/nvim-treesitter/nvim-treesitter/pull/3529) that still not merged to implements treesitter for twig files.

**Comment by numToStr** [comment]
> When I open twig files (without any plugins), neovim set html.twig as filetype.

Interesting, can you try it with `nvim --clean`? I am just curious as I can't find any reference of `html.twig` source code and I certainly don't want to add wrong filetype.

> Syntax highlight doesn't work without plugins in Neovim but I've made a PR that still not merged to implements treesitter for twig files.

Nice 👍

**Comment by numToStr** [comment]
So, I checked it myself, by default `nvim --clean foo.twig` sets filetype to `twig` but there is no highlighting but setting `html.twig` I get highlighting only on html tags.

image

**Comment by gbprod** [comment]
Thanks for the feedback, I've also checked and I found that it's vim-polyglot who set `html.twig` as filetype. I'll update my code tomorrow!

File: lua/Comment/ft.lua [hunk 1]
```diff
@@ -112,6 +112,7 @@ local L = setmetatable({
     template = { M.dbl_hash },
     tmux = { M.hash },
     toml = { M.hash },
+    ['html.twig'] = { '{#%s#}', '{#%s#}' },
```
Comment (numToStr) [inline lua/Comment/ft.lua:115]: Beside filetype, store common commenstring as `M.twig` above and reference it here.

**Review by numToStr — CHANGES_REQUESTED** [review]

File: lua/Comment/ft.lua [hunk 2]
```diff
@@ -112,6 +113,7 @@ local L = setmetatable({
     template = { M.dbl_hash },
     tmux = { M.hash },
     toml = { M.hash },
+    twig = { M.twig,  M.twig },
```
Comment (numToStr) [inline lua/Comment/ft.lua:116]:
```suggestion
    twig = { M.twig, M.twig },
```

File: lua/Comment/ft.lua [hunk 3]
```diff
@@ -27,6 +27,7 @@ local M = {
     latex = '%%s',
     lisp_l = ';;%s',
     lisp_b = '#|%s|#',
+    twig = '{#%s#}'
```
Comment (numToStr) [inline lua/Comment/ft.lua:30]:
```suggestion
    twig = '{#%s#}',
```

**Review by numToStr — APPROVED** [review]

**Pull request merged**